src/openMVG/sfm/sfm_data_io_gt.hpp

```diff
@@ -5,19 +5,24 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-#ifndef IO_READ_GT_HPP
-#define IO_READ_GT_HPP
+#ifndef OPENMVG_SFM_DATA_IO_GT_HPP
+#define OPENMVG_SFM_DATA_IO_GT_HPP
+
+#include "sfm_data_io.hpp"
 
 #include "openMVG/geometry/pose3.hpp"
+#include "openMVG/exif/exif_IO_EasyExif.hpp"
+
 #include "third_party/stlplus3/filesystemSimplified/file_system.hpp"
 
 #include <fstream>
 #include <vector>
 
-namespace openMVG
-{
+namespace openMVG {
+namespace sfm {
 
 using namespace openMVG::cameras;
+using namespace openMVG::exif;
 
 static bool read_openMVG_Camera(const std::string & camName, Pinhole_Intrinsic & cam, geometry::Pose3 & pose)
```

Comment:
- create  a `sfm_data_io_gt.cpp` and move the implementation code there. (in case, there is a ok place to use `using namespace`.

- remove `static`

- check and remove/add the `#include` that are strictly necessary for (compiling) the header, move the others in the `.cpp`